src/modules/sensors/sensors.cpp

```diff
@@ -254,6 +261,10 @@ Sensors::Sensors(bool hil_enabled) :
 	memset(&_diff_pres, 0, sizeof(_diff_pres));
 	memset(&_parameters, 0, sizeof(_parameters));
 
+	_orb_class_instance = -1;
+	_min_sonar_distance = 0.2f;
+	_max_sonar_distance = 7.65f;
```

Comment:
The above should be handled as configurable PX4 parameters (as the sonars may also be different)